src/components/tickets/components/Table.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Avatar, Tag, Table } from 'antd';
+
+import { DATA, NEWTICKET } from '../constants';
+
+const TicketTable = () => (
+    <Table
+        dataSource={DATA}
+        title={() => <b>{ NEWTICKET }</b>}
+        columns={
+            [
+                {
+                    dataIndex: 'userAvatar',
+                    key: 'userAvater',
+                    render: userAvater => <Avatar src={userAvater} />,
+                    title: '',
+                },
+                {
+                    dataIndex: 'ticketDetails',
+                    key: 'ticketDetails',
+                    title: 'Ticket details',
+                },
+                {
+                    dataIndex: 'customerName',
+                    key: 'customerName',
+                    title: 'Customer Name',
+                },
+                {
+                    dataIndex: 'date',
+                    key: 'date',
+                    title: 'Date',
+                },
```

Comment:
push to constants and spread in js file. I will refactor mine so u see how it works